scripts/upload/upload-model.sh

```diff
@@ -78,7 +78,7 @@ TARGET_REWARD_FILE_S3_KEY="s3://${TARGET_S3_BUCKET}/${TARGET_S3_PREFIX}/model/re
 TARGET_HYPERPARAM_FILE_S3_KEY="s3://${TARGET_S3_BUCKET}/${TARGET_S3_PREFIX}/ip/hyperparameters.json"
```

Comment:
The TARGET_REWARD_FILE_S3_KEY also needs to be adapted to move the file into the root, not model.

Reply:
Ah yes, missed this